tools/osx/xcode_configure.bzl

```diff
@@ -151,7 +152,10 @@ def run_xcode_locator(repository_ctx, xcode_locator_src_label):
             out = xcode_locator_result.stdout,
         ).replace("\n", " ")
         print(error_msg)
```

Comment:
Should we still print the error message if there isn't actually a problem?

Reply:
Good point, moved it to the failure branch